egret/model_library/transmission/branch.py

```diff
@@ -944,11 +949,21 @@ def declare_ineq_angle_diff_branch_lbub_c_s(model, index_set, branches):
         from_bus = branches[branch_name]['from_bus']
         to_bus = branches[branch_name]['to_bus']
 
-        m.ineq_angle_diff_branch_lb[branch_name] = (math.tan(math.radians(branches[branch_name]['angle_diff_min'])) *
-                                                    m.c[(from_bus, to_bus)] <= m.s[(from_bus, to_bus)])
-        m.ineq_angle_diff_branch_ub[branch_name] = (m.s[(from_bus, to_bus)] <=
-                                                    math.tan(math.radians(branches[branch_name]['angle_diff_max'])) *
-                                                    m.c[(from_bus, to_bus)])
+        if branches[branch_name]['angle_diff_min'] > -90:
+            if branches[branch_name]['angle_diff_min'] < -89:
+                msg = 'angle difference limits larger than 89 will introduce large coefficients'
+                logger.warning(msg)
+                warnings.warn(msg)
+            m.ineq_angle_diff_branch_lb[branch_name] = (math.tan(math.radians(branches[branch_name]['angle_diff_min'])) *
+                                                        m.c[(from_bus, to_bus)] <= m.s[(from_bus, to_bus)])
+        if branches[branch_name]['angle_diff_max'] < 90:
+            if branches[branch_name]['angle_diff_min'] > 89:
+                msg = 'angle difference limits larger than 89 will introduce large coefficients'
+                logger.warning(msg)
+                warnings.warn(msg)
+            m.ineq_angle_diff_branch_ub[branch_name] = (m.s[(from_bus, to_bus)] <=
+                                                        math.tan(math.radians(branches[branch_name]['angle_diff_max'])) *
+                                                        m.c[(from_bus, to_bus)])
```

Comment:
If the angle min is less than -90 deg, we are skipping the constraint? Is that correct? (same for max and 90 deg)

Reply:
Correct